networking/determine-best-networking.md

```diff
@@ -1,110 +1,219 @@
 ---
 title: Determine best networking option
-description: Calico provides several networking implementations based on IP routing without the need for encapsulation. But encapsulation is supported when you need it.
+description: Learn about the different networking options {{site.prodname}} supports so you can choose the best option for your needs.
 ---
 
 ### Big picture
 
-{{site.prodname}} supports several container networking options for scalability, network performance, and interoperability with existing infrastructure.
+Learn about the different networking options {{site.prodname}} supports so you can choose the best option for your needs.
 
 ### Value
 
-Different network implementations are better suited to different environments. {{site.prodname}} provides several networking implementations based on IP routing without the need for encapsulation. If your deployment requires encapsulation, {{site.prodname}} provides overlay networking (IP in IP or VXLAN). {{site.prodname}} also supports enforcing policy with a number of other Kubernetes networking options. This document helps you choose the best networking option for your cluster.
+{{site.prodname}}’s flexible modular architecture supports a wide range of deployment options, so you can select the best networking approach for your specific environment and needs. This includes the ability to run with a variety of CNI and IPAM plugins, and underlying network types, in non-overlay or overlay modes, with or without BGP.
 
 ### Concepts
 
-#### About {{site.prodname}} networking
+If you want to fully understand the network choices available to you, we recommend you make sure you are familiar with and understand the following concepts.  If you would prefer to skip the learning and get straight to the choices and recommendations, you can jump ahead to [Networking Options](#networking-options).
 
-{{site.prodname}} provides a few ways to allow pods to connect to other pods, to the host, and to outside networks (for example, the internet).
+#### Kubernetes networking basics
+The Kubernetes network model defines a “flat” network in which:
+- every pod get its own IP address
+- pods on any node can communicate with all pods on all other nodes without NAT.
 
-{{site.prodname}} networking:
+This creates a clean, backwards-compatible model where Pods can be treated much like VMs or physical hosts from the perspectives of port allocation, naming, service discovery, load balancing, application configuration, and migration.  Network segmentation can be defined using Network Policies to restrict traffic within this base networking capabilities.
```

Comment:
Lowercase for pods and network policies for Concepts...use full resource name in Features, How to section, and how to examples. 

Network segmentation can be defined using network policies to restrict traffic within this base networking capabilities. (these base networking capabilities?)